pegasus/sites.v3/advocacy.code.org/public/index.haml

```diff
@@ -0,0 +1,58 @@
+---
+theme: responsive_wide
+layout: wide
+social:
+  "og:title": "Advocate for computer science education"
+  "og:description": "Bringing together more than 50 industry, non-profit, and advocacy organizations, the Code.org Advocacy Coalition advances the development of policies to make Computer Science a fundamental part of the K-12 education system."
+  "twitter:site": "@codeorg"
+  "twitter:title": "Advocate for computer science education"
+  "twitter:description": "Learn about our local policy and access resources to support computer science in our state."
+  "twitter:url": "https://advocacy.code.org/"
+video_player: false
+---
+
+#homepage
+
+  #fullwidth{style: "position: relative"}
```

Comment:
What elements are positioned off this element?

Reply:
Ah, `.banner` is positioned `absolute` as a child, but since this is in the top-left corner of the page it seems that this wasn't necessary after all.